app/views/admin/divisions/index.html.slim

```diff
@@ -29,4 +30,6 @@
 
       .clearfix
 
-= render_grid(@divisions_grid)
+.index-grid class="#{'no-records' if @no_records}"
+  = render_grid(@divisions_grid)
```

Comment:
Clearing out the filters does not work unless the grid is loaded on the page. Therefore, the grid is hidden on the page but in the DOM.